packages/cdk/lib/roles/policies/nextflow-describe-jobs-batch-policy.ts

```diff
@@ -0,0 +1,16 @@
+import * as iam from "monocdk/aws-iam";
+
+export class NextflowDescribeJobsBatchPolicy extends iam.PolicyDocument {
+  constructor() {
+    super({
+      assignSids: true,
+      statements: [
+        new iam.PolicyStatement({
+          effect: iam.Effect.ALLOW,
+          actions: ["batch:DescribeJobs"],
+          resources: ["*"],
+        }),
+      ],
+    });
+  }
+}
```

Comment:
Does pulling this out into a class clarify much? Just doing this inline seems just as readable and less code to understand.

`adapter.grant('batch:DescribeJobs', '*')` seems just as good.

Reply:
Sounds good

Reply:
`adapter.grant()` will actually not work sense you don't define the resource in the grant method, but instead in this case the `adapter` will be the resource. I have pulled this definition out of its own class and made it defined within the role document